jenkins/opensearch/check-for-build.jenkinsfile

```diff
@@ -0,0 +1,86 @@
+lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    options {
+        disableConcurrentBuilds() // Make sure not to interupt builds in progress
+    }
+    triggers {
+        parameterizedCron '''
+            H */12 * * * %INPUT_MANIFEST=1.3.0/opensearch-1.3.0.yml;JOB_NAME=distribution-build-opensearch
+            H 1 * * * %INPUT_MANIFEST=2.0.0/opensearch-2.0.0.yml;JOB_NAME=distribution-build-opensearch
+        '''
+    }
+    parameters {
+        string(
+            name: 'INPUT_MANIFEST',
+            description: 'Input manifest under the manifests folder, e.g. 2.0.0/opensearch-2.0.0.yml.',
+            trim: true
+        )
+        string(
+            name: 'JOB_NAME',
```

Comment:
Jenkins has an env variable called `JOB_NAME` for the current job that is running. Will this overwrite that or how will it work? Should we change it to something little less generic?

Reply:
Good idea, lets go with `TARGET_JOB_NAME`